Makefile

```diff
@@ -17,3 +17,6 @@ fmt:
 clean:
 	rm -f ${APP_NAME}; \
 	docker rmi -f ${IMGNAME}:latest
+
+lint:
+    golint ./...
```

Comment:
Small fix here. This should be prefaced with a tab to satisfy `make` and match the other lines.